app/globals.css

```diff
@@ -3,7 +3,8 @@
 @tailwind utilities;
 
 html,body, :root {
-  height: 100;
+  height: 100%;
+  background-color: red;
```

Comment:
Remove this background color, does not look good.

Reply:
was testing git